File: c/7.c
/*
    main 함수 안의 내용만 수정해주세요.
    수정할 경우, 자동 채점 프로그램이 제대로 동작하지 않을 가능성이 있습니다.
*/

#include <stdio.h>

int main() {
    // 이곳에 코드를 작성해주세요!
int getNumberOfDaysInMonth(int year, int month) {
    if (month == 1 || month == 3 || month == 5 || month == 7 || month == 8 || month == 10 || month == 12) return 31;
    if (month == 4 || month == 6 || month == 9 || month == 11) return 30;
    if (month == 2) {
        if (isLeapYear(year) == 1) return 29;
        else return 28;
    }
    int isLeapYear(int year) {
	else return 0;
}
    return 0;
}

File: python/7.py
"""
    main 함수 안의 내용만 수정해주세요.
    수정할 경우, 자동 채점 프로그램이 제대로 동작하지 않을 가능성이 있습니다.
"""
def main():
    def is_leap_year(year):
        if (year % 4 == 0 and year % 100 != 0) or (year % 400 == 0):
            return True
        else:
            return False

    def days_in_month(year, month):
        if month in [4, 6, 9, 11]:
            return 30
        elif month == 2:
            if is_leap_year(year):
                return 29
            else:
                return 28
        else:
            return 31
        
    year = int(input("연도를 입력하세요: "))
    month = int(input("월을 입력하세요: "))

    days = days_in_month(year, month)
    print(days)

if __name__ == "__main__":
    main()